astroquery/mast/core.py

```diff
@@ -16,10 +16,7 @@
 
 import numpy as np
 
-try: # Python 3.x
-    from urllib.parse import quote as urlencode
-except ImportError:  # Python 2.x
-    from urllib import pathname2url as urlencode
+from six.moves.urllib.parse import quote as urlencode
```

Comment:
Please use the astropy bundled six (as we don't have it as dependency for astroquery):

``from astropy.extern import six``